app/views/shared/_user_component_certificates.html.erb

```diff
@@ -25,9 +25,15 @@
             </td>
           </tr>
         <% end%>
-        <% component.signing_certificates.each do |signing| %>
+        <% component.signing_certificates.reverse.each do |signing| %>
           <tr class="govuk-table__row">
-            <td class="govuk-table__cell"><%= link_to t('user_journey.signing_certificate') , view_certificate_path(signing.component_type, component.id, signing.id) %></td> 
+            <td class="govuk-table__cell">
+              <% if component.signing_certificates.length == 2 %>
+                <%= link_to t('user_journey.two_signing_certificate', type: primary_signing_certificate(signing) ? t('user_journey.primary') : t('user_journey.secondary')) , view_certificate_path(signing.component_type, component.id, signing.id) %>
```

Comment:
We could make this shorter and remove the `if` statement by having the model returning the position as a string
```suggestion
                <%= link_to t('user_journey.two_signing_certificate', type: t("user_journey.#{position(signing)")) , view_certificate_path(signing.component_type, component.id, signing.id) %>
```
similar down below